zer0-os/zOS
# feat(post-input): create and add new post input component with media selection functionality

### What does this do?
- removes existing CreatePost component
- adds new PostInput component
- handles media selection functionality

### Why are we making this change?
- improve post input component to implement additional functionality such as selecting and adding media (images)

### How do I test this?
- run tests as usual.
- run UI as usual > open/create social channel > check post input is working as expected i.e. sending text.
- NOTE :: sending images has not yet been wired up, for now, this input will only send text 

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?


https://github.com/user-attachments/assets/85e49cab-60af-4053-aec7-dfefceae90b2
